theme.json

```diff
@@ -0,0 +1,140 @@
+{
+	"$schema": "https://schemas.wp.org/trunk/theme.json",
```

Comment:
It's best to align the `theme.json` schema with the current version of WordPress.